### DISCUSSION TITLE
不显示文件，提示：failed get objs: failed to list objs: TooManyRequests:Throttled by PLUGIN Flow Control

### DISCUSSION THREAD

**Discussion by starplm** [AlistGo/alist#4591] [General]
提示：failed get objs: failed to list objs: TooManyRequests:Throttled by PLUGIN Flow Control

**Comment by bzzjh** [comment]
我也同样问题，被流控了，是因为不是阿里会员吗

**Reply by bzzjh** [reply]
我是在备份文件，很多上传

**Comment by 1273082756** [comment]
同问

**Comment by liuming946** [comment]
我的阿里云盘open挂在也有通同样的问题，在文件夹中超过1000个文件后会报这个错误TooManyRequests:Throttled by PLUGIN Flow Control。这个API是否可以分页查询，然后合并返回给页面或DAV。如果已经分页了，是否是因为访问频率过快的问题，可以考虑增加延迟时间，以请求到完整的目录。

**Reply by paomianshaokao** [reply]
alist存储配置中，需要开启Web代理

**Comment by kele-manong** [comment]
我也有这个问题，我显示文件清单时并不会触发，而是在复制了下载链接后，用aria2下载的时候，有概率触发，下载下来的文件115b大小，打开里面就这段话。
目前还不知道怎么解决比较好

**Comment by jibwf** [comment]
同样的问题，好像最近才遇到的。
我是用alist配合rclone备份，当文件多的时候，就会遇到这个问题。

**Reply by kele-manong** [reply]
我是下载遇到这个问题，本质是网盘的链接数量达到限制，我的解决方法非常粗暴，就是将aria2的连接数量降低到1，一次只下载一个文件，就可以避免这个问题发生，当然这本质上是个笨办法，对于我下载来说，最终的下载速度实际上没影响（单个文件还是可以开多个线程），就是每次就一个文件，如果大量小文件，那么性能不太好。
